templates/appliance/adguard/intel-nuc.html

```diff
@@ -10,6 +10,6 @@
 
 {% include 'appliance/shared/_intel-nuc.html' %}
 
-{% include 'appliance/adguard/_start-using.html' %}
+{% include 'appliance/'+appliance.url_name+'/_next-steps.html' %}
```

Comment:
I wonder if f-strings work in templates?

```suggestion
{% include f"appliance/{appliance.url_name}/_next-steps.html" %}
```

Reply:
not to f strings; however, it does seem to like `~` to do the same - https://jinja.palletsprojects.com/en/2.11.x/templates/